{
  "status": "SUCCESS",
  "cpu_usage": [
    "27"
  ],
  "stdout": [
    "EPI(k=10,i=  50): cost -7.43E+00, H  7.67E+00, |R|^2 1.76E-02\n",
    "EPI(k=10,i= 100): cost -7.58E+00, H  7.63E+00, |R|^2 1.77E-02\n",
    "EPI(k=10,i= 150): cost -7.46E+00, H  7.63E+00, |R|^2 8.31E-03\n",
    "EPI(k=10,i= 200): cost -7.48E+00, H  7.68E+00, |R|^2 2.15E-02\n",
    "EPI(k=10,i= 250): cost -7.48E+00, H  7.73E+00, |R|^2 1.58E-02\n",
    "EPI(k=10,i= 300): cost -7.61E+00, H  7.69E+00, |R|^2 7.86E-03\n",
    "EPI(k=10,i= 350): cost -7.52E+00, H  7.66E+00, |R|^2 1.09E-02\n",
    "EPI(k=10,i= 400): cost -7.57E+00, H  7.56E+00, |R|^2 1.52E-02\n",
    "EPI(k=10,i= 450): cost -7.47E+00, H  7.67E+00, |R|^2 1.98E-02\n",
    "EPI(k=10,i= 500): cost -7.49E+00, H  7.77E+00, |R|^2 3.09E-02\n",
    "EPI(k=10,i= 550): cost -7.48E+00, H  7.68E+00, |R|^2 1.72E-02\n",
    "EPI(k=10,i= 600): cost -7.56E+00, H  7.60E+00, |R|^2 4.92E-03\n",
    "EPI(k=10,i= 650): cost -7.49E+00, H  7.72E+00, |R|^2 1.68E-02\n",
    "EPI(k=10,i= 700): cost -7.42E+00, H  7.63E+00, |R|^2 1.41E-02\n",
    "EPI(k=10,i= 750): cost -7.35E+00, H  7.60E+00, |R|^2 1.48E-02\n",
    "EPI(k=10,i= 800): cost -7.48E+00, H  7.42E+00, |R|^2 5.49E-03\n",
    "EPI(k=10,i= 850): cost -7.25E+00, H  7.61E+00, |R|^2 3.44E-02\n",
    "EPI(k=10,i= 900): cost -7.39E+00, H  7.50E+00, |R|^2 1.27E-02\n",
    "EPI(k=10,i= 950): cost -7.36E+00, H  7.38E+00, |R|^2 2.67E-02\n",
    "EPI(k=10,i=1000): cost -7.36E+00, H  7.52E+00, |R|^2 1.55E-02\n"
  ],
  "instance": "i-0190156fe398b320c",
  "reason": [
    "aws s3 sync ./ \"s3://$bucketname/$groupdir/$resultdir/per_hp\""
  ],
  "command": "ac755bab-08ff-4a3e-891c-d83126c884b7",
  "stderr": [
    "2020-03-10 14:02:12.506025: W tensorflow/stream_executor/platform/default/dso_loader.cc:55] Could not load dynamic library 'libnvinfer.so.6'; dlerror: libnvinfer.so.6: cannot open shared object file: No such file or directory\n",
    "2020-03-10 14:02:12.506101: W tensorflow/stream_executor/platform/default/dso_loader.cc:55] Could not load dynamic library 'libnvinfer_plugin.so.6'; dlerror: libnvinfer_plugin.so.6: cannot open shared object file: No such file or directory\n",
    "2020-03-10 14:02:12.506109: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:30] Cannot dlopen some TensorRT libraries. If you would like to use Nvidia GPU with TensorRT, please make sure the missing libraries mentioned above are installed properly.\n",
    "2020-03-10 14:02:47.138607: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcuda.so.1\n",
    "2020-03-10 14:02:47.619009: E tensorflow/stream_executor/cuda/cuda_driver.cc:351] failed call to cuInit: CUDA_ERROR_NO_DEVICE: no CUDA-capable device is detected\n",
    "2020-03-10 14:02:47.619046: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:156] kernel driver does not appear to be running on this host (ip-172-31-68-92): /proc/driver/nvidia/version does not exist\n",
    "2020-03-10 14:02:47.619580: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 AVX512F FMA\n",
    "2020-03-10 14:02:47.699049: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 2499995000 Hz\n",
    "2020-03-10 14:02:47.699896: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x55dd6b711420 initialized for platform Host (this does not guarantee that XLA will be used). Devices:\n",
    "2020-03-10 14:02:47.699917: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version\n",
    "WARNING:tensorflow:From /home/ubuntu/anaconda3/envs/epi/lib/python3.6/site-packages/tensorflow_probability/python/bijectors/real_nvp.py:386: dense (from tensorflow.python.layers.core) is deprecated and will be removed in a future version.\n",
    "Instructions for updating:\n",
    "Use keras.layers.Dense instead.\n",
    "WARNING:tensorflow:From /home/ubuntu/anaconda3/envs/epi/lib/python3.6/site-packages/tensorflow_core/python/layers/core.py:187: Layer.apply (from tensorflow.python.keras.engine.base_layer) is deprecated and will be removed in a future version.\n",
    "Instructions for updating:\n",
    "Please use `layer.__call__` method instead.\n",
    "WARNING:tensorflow:From /home/ubuntu/anaconda3/envs/epi/lib/python3.6/site-packages/tensorflow_probability/python/bijectors/real_nvp.py:222: AffineScalar.__init__ (from tensorflow_probability.python.bijectors.affine_scalar) is deprecated and will be removed after 2020-01-01.\n",
    "Instructions for updating:\n",
    "`AffineScalar` bijector is deprecated; please use `tfb.Shift(loc)(tfb.Scale(...))` instead.\n"
  ],
  "input": "cunninghamlabEPI/inputs/epidata/dataset10.json"
}
